src/libs/algolia/fetchAlgolia/fetchSearchResults/fetchSearchResults.ts

```diff
@@ -142,12 +142,18 @@ export const fetchSearchResults = async ({
     },
     // Artists in offers
     {
-      ...offersQuery,
+      indexName: offersIndex,
+      query: '',
       params: {
-        ...offersQuery.params,
+        ...buildOfferSearchParameters(
+          { ...parameters, query: '', artistName: parameters.query },
```

Comment:
```suggestion
          { ...parameters, artistName: parameters.query },
```
`query` n'est pas utilisée dans `buildOfferSearchParameters`, donc normalement pas besoin de l'override ici